words_dir/tweet_nouns.py

```diff
@@ -8,6 +8,7 @@
 # NOTE: nouns should be singular (e.g. 'boot' not 'boots')
 # NOTE: must make sense in "{adjective} {label} {noun}" (example: tiny queer [something])
 # NOTE: must fit in a tweet, so not longer than this line -----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+# NOTE: if it can be used as a name for an enby, its an awesome noun
```

Comment:
While I appreciate this note, it doesn't add to the project. If you would like to add nouns like this, you're absolutely welcome to and I would love it if you did. There's a couple of lists at the bottom of the file for `queer things`